maas/plugins/hp_monitoring.py

```diff
@@ -59,7 +59,7 @@ def get_controller_status():
 
 def get_controller_cache_status():
     return check_command(('hpssacli', 'ctrl', 'all', 'show', 'status'),
-                         'Cache Status', 'OK')
+                         'Cache Status', ('OK', 'Not Configured'))
```

Comment:
OK lets add an override so we can allow unconfigured cache of alternatively we can check for installed drives with 'hpssacli ctrl slot=3 pd all show' and search for 'Solid State' turning off the cache